tools/WinObjC.Packaging/WinObjC.Packager.targets

```diff
@@ -45,6 +40,17 @@
     </ItemGroup>
   </Target>
 
+  <!-- 
+    In the above AddSingleConfigBuildOutput target, several properties are injected into this project so that this project knows if its being multiplexed or not. Don't pass on this extra information to project references as it can issues with MSBuild knowing if a project is already built.
+  -->
```

Comment:
nit: make this comment multiple lines. I think a couple other places have overrun comments as well. 
also add a word: "[...] project references as it can _have_ issues with MSBuild [...]"